cloud/endagaweb/views/dashboard.py

```diff
@@ -560,6 +560,9 @@ def post(self, request, imsi=None):
                     CURRENCIES[currency]).amount_raw
             if abs(amount) > 2147483647:
                 raise ValueError(error_text)
+            if sub.balance + amount > network.max_account_limit:
+                error_text = 'Error : Crossed Credit Limit.'
```

Comment:
Now while checking cross limitation ,there also take out network_max_balance based on currency.written unit test case.
comments incorporated in commit id :1e5832b491ca904b5fd002d9d61f3d1fca09cbcf.
attached console output for unit test:

![unittest_for_network_max_balance](https://user-images.githubusercontent.com/26892520/26941517-5e40a65c-4c9c-11e7-9860-660e9102a38f.png)